kubernetes/services/ignite_config.xml

```diff
@@ -43,6 +43,100 @@ Copyright(c) 2020 Futurewei Cloud
+                 <property name="name" value="com.futurewei.alcor.dataplane.entity.InternalSubnets"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                 <property name="name" value="com.futurewei.alcor.dataplane.entity.InternalSubnetPorts"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                 <property name="name" value="com.futurewei.alcor.dataplane.entity.GatewayInfo"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                <property name="name" value="ncm_nodeinfo_cache"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                 <property name="name" value="com.futurewei.alcor.netwconfigmanager.entity.ResourceMeta"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                 <property name="name" value="com.futurewei.alcor.netwconfigmanager.entity.VpcResourceMeta"/>
+                <!-- Set the cache mode. -->
+                <property name="atomicityMode" value="TRANSACTIONAL"/>
+                <!-- Other cache parameters. -->
+                <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                <!-- Set the cache name. -->
+                 <property name="name" value="java.lang.Object"/>
```

Comment:
Yes, that is because of the cache/ResourceStateCache.java:
public class ResourceStateCache {
    // Map <ResourceId, ResourceState>
    private ICache<String, Object> hostResourceStates;
...
}

.

Reply:
Cache name = `java.lang.Object` is kind of dangerous!

Reply:
I agree but Liguang has to make the change here. Perhaps, class SomeObject { Object obj }, and use SomeObject as the value type.